### PR TITLE
Updating Class Names

### PR DESCRIPTION
**Pull request by ankursinghchauhan10**

### CODE REVIEW

File: src/problem1/Test_VCS_Repository.java [hunk 1]
```diff
@@ -5,13 +5,13 @@
 import org.junit.BeforeClass;
```
Comment (bsmaheshkumar) [inline src/problem1/Test_VCS_Repository.java:5]: Next time
Please raise a pull request...
Dont commit the code directly...

File: src/problem1/Test_VCS_Repository.java [hunk 2]
```diff
@@ -5,13 +5,13 @@
...
 import org.junit.Test;
 
-public class TestFile {
+public class Test_VCS_Repository {
```
Comment (neerajkujha) [inline src/problem1/Test_VCS_Repository.java:8]: you can use proper javadoc and comments for test classes as well

File: src/problem1/Test_VCS_Repository.java [hunk 3]
```diff
@@ -28,7 +28,7 @@ public void TestVCSSystem() {
 		Integer B[] = { 3, 4 };
 		String expected = "0 , 0";*/
 
-		String output = ob.VCS(N, M, K, A, B);
+		String output = ob.getAllFileCount(N, M, K, A, B);
 		
 		//Output of the VCS Function
 		System.out.println(output);
```
Comment (neerajkujha) [inline src/problem1/Test_VCS_Repository.java:34]: sysout should not be used

**Review by neerajkujha — COMMENTED** [review]
javadoc should be applied to UT as well. Method doesn't have proper javadoc. Negative Test cases needs to added for all tests

**Pull request merged**